app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonFilters.kt

```diff
@@ -0,0 +1,140 @@
+            Text(
+                stringResource(R.string.dashboard_lessons_section_filter_label),
+                modifier = Modifier
+                    .align(Alignment.CenterVertically)
+                    .padding(end = 36.dp)
+            )
+            LessonLanguageFilter(
+                modifier = Modifier
+                    .weight(1f)
+                    .align(Alignment.CenterVertically)
+            )
+        }
+    }
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalFoundationApi::class)
+@Composable
+internal fun LessonLanguageFilter(modifier: Modifier = Modifier, viewModel: LessonsViewModel = viewModel()) {
+    val context = LocalContext.current
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState()
+
+    val lessonLanguages by viewModel.filteredLanguages.collectAsState(emptyList())
+    var expanded by remember { mutableStateOf(false) }
+    var query by remember { mutableStateOf("") }
+    val scope = rememberCoroutineScope()
+
+    ElevatedButton(
+        onClick = { expanded = true },
+        modifier = modifier.semantics { role = Role.DropdownList }
+    ) {
+        Text(
+            selectedLanguage.getDisplayName(context, LocalAppLanguage.current),
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            modifier = Modifier.weight(1f),
+        )
+        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded)
+        LazyDropdownMenu(
+            expanded = expanded,
+            onDismissRequest = { expanded = false },
+            modifier = Modifier.sizeIn(maxHeight = DROPDOWN_LESSON_MAX_HEIGHT, maxWidth = DROPDOWN_LESSON_MAX_WIDTH)
+        ) {
+            stickyHeader {
+                Surface(color = MaterialTheme.colorScheme.surface) {
+                    SearchBar(
+                        query = query,
+                        onQueryChange = { change ->
+                            scope.launch {
+                                query = change
+                                viewModel.updateQuery(change)
+                            }
+                        },
+                        onSearch = {},
+                        active = false,
+                        onActiveChange = {},
+                        colors = GodToolsTheme.searchBarColors,
+                        placeholder = {
+                            Text(stringResource(R.string.language_settings_downloadable_languages_search))
+                        },
+                        content = {},
+                        modifier = Modifier
+                            .padding(horizontal = 16.dp, vertical = 16.dp)
+                            .fillMaxWidth()
+                            .wrapContentWidth()
+                    )
+                }
+            }
+
+            items(lessonLanguages, key = { (lang) -> lang.code }) { (lang, count) ->
+                HorizontalDivider(Modifier.padding(start = ListItemStartPadding, end = ListItemStartPadding))
```

Comment:
There should be no horizontal divider between the searchbar and the first language

One of these is the current language filter from the tools UI, the other is your new Lessons filter drop down
![Screenshot 2024-08-12 at 3 43 27 PM](https://github.com/user-attachments/assets/f637ca00-0e56-44fe-831b-b0124c4f5bc9)
![Screenshot 2024-08-12 at 3 43 19 PM](https://github.com/user-attachments/assets/6ef79d19-0aad-4ef8-a7f4-460f8f673097)

Reply:
these screenshots also show the issue I mentioned with having extra vertical whitespace